cleansweep/views/admin.py

```diff
@@ -283,3 +283,18 @@ def _load_contacts(place, data):
         contacts += p.add_contacts(prows)
     db.session.commit()
     return contacts
+
+
+@app.route("/admin/settings/export", methods=['GET'])
```

Comment:
> Are we gonna have separate endpoint for each export?
>
> I was thinking to do it just once in admin settings. Like mentioned in
> #182 <https://github.com/AamAadmiParty/cleansweep/issues/182>.
>

Even if we choose to have a single export of everything, it might be good
to have individual exports.

The right way to have single export option work without having the import
the plugins is by maintaining a registry. I'll add a detailed note about
this a little later.

For now, could you please move that code to committees and add individual
exports?